### PR TITLE
[Docs IA][ESS] Rework the Elastic Defend and endpoint protection sections

### PR DESCRIPTION
**Pull request by joepeeples**
Contributes to https://github.com/elastic/security-docs/issues/5662: reorganizes the Defend and endpoint protection content in the ESS docs.

_There are still plenty of gaps, on plenty of pages, but we can spin off additional PRs to create and revise content for landing pages, introductions, etc. The main goal of this PR is to rebuild the structure for that content._

## Previews
Lots of pages are moved around. Everything is now in these three main sections, which are reordered next to each other:

- [Endpoint protection with Elastic Defend](https://security-docs_bk_5707.docs-preview.app.elstc.co/guide/en/security/master/endpoint-protection-intro.html)
- [Manage Elastic Defend](https://security-docs_bk_5707.docs-preview.app.elstc.co/guide/en/security/master/sec-manage-intro.html)
- [Endpoint response actions](https://security-docs_bk_5707.docs-preview.app.elstc.co/guide/en/security/master/response-actions.html)

You might also want to check out the "Get started" section, now that a lot of Defend/Endpoint content has been moved out. Looks a lot cleaner:

- [Get started with Elastic Security](https://security-docs_bk_5707.docs-preview.app.elstc.co/guide/en/security/master/getting-started.html)

### Rationale

In our planning, we considered the original proposal for a single section for everything, but it seemed overloaded and had problems with AsciiDoc's nesting limits. Alternate draft PR https://github.com/elastic/security-docs/pull/5709 shows what a single-section IA looks like, for A/B comparison.

### Summary of changes
Overall, we're emphasizing `Elastic Defend` instead of scattered, non-deliberate references to `Endpoint`. In most cases, customers don't need to know or think about Elastic Endpoint, so we usually don't need to mention it. Of course, we're not avoiding `Endpoint` when it's truly relevant, so there are still a few select, deliberate references.

Also, some generic references to lowercase `endpoint` or `endpoint protection` aren't always necessary or helpful, because we only have one endpoint protection solution, and it's Elastic Defend.

- Reorg & reorder into three sections.
- Rename sections to emphasize Elastic Defend (such as `Manage endpoint protection` → `Manage Elastic Defend`).
- Touch up limited content to improve context-setting and explanations.
- Combine `Uninstall Elastic Agent` and `Uninstall Elastic Endpoint`.
- Create new topic `Troubleshoot Elastic Defend` (technically just moving existing troubleshooting content).
- Remove Endgame installation (Endgame's docs can cover Endgame).

### Serverless twin

I'm paranoid about ESS and serverless docs getting out of sync, especially with all the changes made in this PR, so I went ahead and opened a serverless twin PR:

- https://github.com/elastic/security-docs/pull/5715

### CODE REVIEW

**Review by benironside — DISMISSED** [review]
This version looks great to me. Compared to the all-in-one-section version, I think the organization is intuitive, and each section's topic matches its contents well, which should make each page more findable. I like the minor content adjustments to help clarify the relationship between Defend and Endpoint. All in all, kudos, thanks for wrangling the indices, I think this would be a big improvement.

File: docs/management/admin/endpoint-self-protection.asciidoc [hunk 1]
```diff
@@ -25,5 +25,5 @@ For {stack} version >= 7.11.0, self-protection defines the following permissions
 
 * Users -- even Administrator/root -- *cannot* delete {elastic-endpoint} files (located at `c:\Program Files\Elastic\Endpoint` on Windows, and `/Library/Elastic/Endpoint` on macOS).
 * Users *cannot* terminate the {elastic-endpoint} program or service.
-* Administrator/root users *can* read the Endpoint's files. On Windows, the easiest way to read Endpoint files is to start an Administrator `cmd.exe` prompt. On macOS, an Administrator can use the `sudo` command.
+* Administrator/root users *can* read the {elastic-endpoint}'s files. On Windows, the easiest way to read {elastic-endpoint} files is to start an Administrator `cmd.exe` prompt. On macOS, an Administrator can use the `sudo` command.
```
Comment (natasha-moore-elastic) [inline docs/management/admin/endpoint-self-protection.asciidoc:28]:
```suggestion
* Administrator/root users *can* read {elastic-endpoint}'s files. On Windows, the easiest way to read {elastic-endpoint} files is to start an Administrator `cmd.exe` prompt. On macOS, an Administrator can use the `sudo` command.
```

**Review by natasha-moore-elastic — APPROVED** [review]
Awesome job, I love how much cleaner the Get started section looks! Thanks for bringing this to life, @joepeeples! 🚀

**Comment by nastasha-solomon** [comment]
These changes look great, thanks @joepeeples! The Defend docs' increased visibility is nice, and I like the flatter hierarchy that multiple sections provide. Thanks as well for replacing Elastic Endpoint with Elastic Defend where it made sense. It's v easy to confuse or conflate the two, so the  more we can disambiguate them, the better.

**Review by nastasha-solomon — APPROVED** [review]

File: docs/getting-started/install-elastic-endpoint-ven.asciidoc [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+= Enable access for macOS Ventura and higher
 
 :frontmatter-description: Manually install and deploy Elastic Endpoint on macOS Ventura and higher.
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [secure]
 
 
-To properly install and configure {elastic-endpoint} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint} can be fully functional:
+To properly install and configure {elastic-defend} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint}—the installed component that performs {elastic-defend}'s threat monitoring and prevention—is fully functional:
```
Comment (joepeeples) [inline docs/getting-started/install-elastic-endpoint-ven.asciidoc:10]:
```suggestion
To properly install and configure {elastic-defend} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the host before {elastic-endpoint}—the installed component that performs {elastic-defend}'s threat monitoring and prevention—is fully functional:
```

File: docs/getting-started/install-elastic-endpoint.asciidoc [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+= Enable access for macOS Monterey
 
-:frontmatter-description: Manually install and deploy Elastic Endpoint on macOS Catalina through Monterey.
+:frontmatter-description: Manually install and deploy Elastic Endpoint on macOS Monterey.
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [secure]
 
-To properly install and configure {elastic-endpoint} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint} can be fully functional:
+To properly install and configure {elastic-defend} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the endpoint before {elastic-endpoint}—the installed component that performs {elastic-defend}'s threat monitoring and prevention—is fully functional:
```
Comment (joepeeples) [inline docs/getting-started/install-elastic-endpoint.asciidoc:9]:
```suggestion
To properly install and configure {elastic-defend} manually without a Mobile Device Management (MDM) profile, there are additional permissions that must be enabled on the host before {elastic-endpoint}—the installed component that performs {elastic-defend}'s threat monitoring and prevention—is fully functional:
```

**Review by jmikell821 — COMMENTED** [review]
Just had one slight suggestion but I am loving these condensed sections! Thanks @joepeeples and @natasha-moore-elastic! 🌟

**Review by benironside — APPROVED** [review]

**Pull request merged**